tests/test_maxStopProcessCount.py

```diff
@@ -3,12 +3,12 @@
 import pytest
 from fasttrips import Run
 
-EXAMPLE_DIR    = os.path.join(os.getcwd(), 'fasttrips', 'Examples', 'Springfield')
```

Comment:
It looks like the changes to the Springfield 'A' config are not part of this (increasing cost_to_util), but I think we should keep these two changes together for clarity

Reply:
And they are on master already, my bad.